src/App.js

```diff
@@ -1,28 +1,9 @@
-import logo from './logo.svg';
+import Calculator from './components/Calculator';
 import './App.css';
 
 function App() {
```

Comment:
- [ ] As it is a requirement for this project: [Use class based components](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md), kindly change your `App.js` component, which is presently  a `functional` component to a `class-based` component.